onnxruntime/core/providers/openvino/ov_interface.cc

```diff
@@ -103,16 +126,29 @@ void OVCore::SetCache(std::string cache_dir_path) {
 }
 
 #ifdef IO_BUFFER_ENABLED
-OVExeNetwork OVCore::LoadNetwork(std::shared_ptr<OVNetwork>& model, OVRemoteContextPtr context, std::string& name) {
+OVExeNetwork OVCore::CompileModel(std::shared_ptr<const OVNetwork>& model, OVRemoteContextPtr context, std::string& name) {
   try {
     auto obj = oe.compile_model(model, *context);
+    printDebugInfo(obj);
     return OVExeNetwork(obj);
   } catch (const Exception& e) {
     ORT_THROW(log_tag + " Exception while Loading Network for graph: " + name + e.what());
   } catch (...) {
     ORT_THROW(log_tag + " Exception while Loading Network for graph " + name);
   }
 }
+OVExeNetwork OVCore::ImportModel(std::istringstream& model_stream, OVRemoteContextPtr context, std::string& name) {
+  try {
+    auto obj = oe.import_model(model_stream, *context);
```

Comment:
Maybe we should check the status of stringstream object here before import to OpenVINO

Reply:
OpenVINO has its own checks on input stream objects, which we catch and re throw if there is an exception in the import_model call.